includes/utils.inc

```diff
@@ -1242,6 +1229,37 @@ function wf_crm_get_fields($var = 'fields') {
           'empty_option' => t('None'),
         );
       }
+      // Membership Date Rule Fields
+      $fields['membership_start_date_rules'] = array(
+        'name' => t('Start Date'),
+        'type' => 'select',
+        'expose_list' => TRUE,
+        'value' => 0,
+        'exposed_empty_option' => t('Automatic'),
+      );
+      $fields['membership_start_date_memberships'] = array (
+        'name' => t('Select Membership Types'),
+        'type' => 'select',
+        'expose_list' => TRUE,
+        'extra' => array('multiple' => 1),
+      );
+      $fields['membership_end_date_rules'] = array(
+        'name' => t('End Date'),
+        'type' => 'select',
+        'expose_list' => TRUE,
+        'value' => 0,
+        'exposed_empty_option' => t('Automatic'),
+      );
+      $fields['membership_end_date_memberships'] = array (
+        'name' => t('Select Membership Types'),
+        'type' => 'select',
+        'expose_list' => TRUE,
+        'extra' => array('multiple' => 1),
+      );
+      $fields['membership_pro_rate_membership'] = array(
```

Comment:
since this field is not related to the commit, it should not be here, and while it is fine for this PR, I think it is important to not do that when you open the PR against webform_civicrm core repo.